network/wsNetwork.go

```diff
@@ -772,6 +777,14 @@ func (wn *WebsocketNetwork) checkServerResponseVariables(header http.Header, add
 	return true
 }
 
+// WaitAndAddConnectionTime will wait to prevent exceeding connectionsRateLimitingCount.
+// Then it will register the next connection time.
+func (wn *WebsocketNetwork) WaitAndAddConnectionTime(addr string) {
```

Comment:
I'm not sure if you really need this function at all. For sure there is no reason to export it.
Deleting this one would save you from implementing it across all the testing interfaces..